packages/build/tests/telemetry/tests.js

```diff
@@ -72,66 +75,91 @@ const runWithApiMock = async function (
   }
 
   try {
-    await runFixture(t, fixture, {
+    const { exitCode } = await runFixture(t, fixture, {
+      useBinary,
       flags: { siteId: 'test', testOpts, ...flags },
       env,
       snapshot,
     })
-  } finally {
+    await Promise.all(stopServers.map((stopServer) => stopServer()))
```

Comment:
Should we keep it `finally` to avoid duplicating this line?
We can still return `{ exitCode, telemetryRequests }` inside the `try{}` catch.

Reply:
Good catch (pun intended 👀), addressed in https://github.com/netlify/build/pull/2415/commits/60db5a2a66339a3e76d4f356fd4a4344bce18ef3